easybuild/easyblocks/p/parmetis.py

```diff
@@ -166,20 +167,33 @@ def install_step(self):
         # other applications depending on ParMETIS (SuiteSparse for one) look for both ParMETIS libraries
         # and header files in the Lib directory (capital L). The following symlink are hence created.
         try:
-            llibdir = os.path.join(self.installdir, 'Lib')
-            os.symlink(libdir, llibdir)
-            for f in ['metis.h', 'parmetis.h']:
-                os.symlink(os.path.join(includedir, f), os.path.join(libdir, f))
+            caplibdir = os.path.join(self.installdir, 'Lib')
+            if os.path.lexists(caplibdir):
```

Comment:
@lexming remove_file() already does this check, so it's not needed.

Reply:
Thanks for pointing this out, fixed :+1: